### ISSUE TITLE
Unknown switch: 0B1B

### ISSUE THREAD

**Issue by alisonrag** [OpenKore/openkore#3005]
based in #2998 

* **Openkore version git**: 
lastest

* **Server**: 
cRO

* **Bug Report / Feature Request**:
Packet Tokenizer: Unknown switch: 0B1B
Inventory Item Protect

* **Summary**:
Server newly updated , and some Unknown switches showed  
Packet Tokenizer: Unknown switch: 0B1B
Request to implement new column of inventory, in this column item can be protected to avoid selled to npc.

![image](https://user-images.githubusercontent.com/34257256/70975233-783b3580-20e4-11ea-983a-c7c5a0b486b7.png)

**Comment by mcpchen** [comment]
CRO is not work
I download Latest commit 2035f38 Jan 2, 2020
command "i" Prompt Inventory is not available
Packet Tokenizer: Unkown switch:0B1B
Use ALT+E Inventory is empty

**Comment by mcpchen** [comment]
Please help update the cRO
According to the operation of the 2998, cRO or cannot be used

**Comment by alisonrag** [comment]
just add into tables/cRO/recvpackets.txt
```
0B00 8 8 1
0B01 56 56 0
0B02 26 26 0
0B03 -1 47 0
0B04 90 90 0
0B05 63 63 1
0B07 -1 47 0
0B08 -1 5 1
0B09 -1 5 1
0B0A -1 5 1
0B0B 4 4 1
0B0C 155 155 1
0B0D 10 10 1
0B0E -1 4 1
0B0F -1 4 1
0B10 10 10 1
0B11 4 4 1
0B12 2 2 -1
0B13 48 48 0
0B14 2 2 0
0B15 7 7 0
0B16 2 2 0
0B17 3 3 0
0B18 4 4 0
0B19 2 2 0
0B1A 29 29 1
0B1B 2 2 1
0B1C 2 2 0
0B1D 2 2 0
0B1E 14 14 0
0B1F 14 14 1
0B20 271 271 1
0B21 13 13 0
0B22 5 5 0
0B23 6 6 0
0B24 6 6 1
0B25 6 6 1
0B27 -1 4 1
0B28 3 3 1
0B2B 11 11 1
0B2C 3 3 1
0B2D 11 11 1
0B2E 4 4 1
0B2F 73 73 1
0B30 -1 4 1
0B31 17 17 1
0B32 -1 4 1
0B33 17 17 1
0B34 50 50 0
0B35 3 3 0
0B36 -1 14 1
0B37 -1 47 1
0B38 -1 28 1
0B39 -1 5 1
0B3A 4 4 1
0B3B 4 4 1
0B3C 4 4 1
0B3D -1 12 1
0B3E -1 8 1
0B3F 64 64 1
0B40 -1 8 1
0B41 41 41 1
0B42 30 30 1
0B43 47 47 1
0B44 32 32 1
0B45 32 32 1
0B46 10 10 0
0B47 14 14 1
0B48 18 18 0
0B49 4 4 0
0B4A 6 6 0
0B4B 4 4 0
0B4C 2 2 0
0B4D -1 6 0
0B4E -1 6 1
0B4F 2 2 0
0B50 2 2 0
0B51 2 2 0
0B52 2 2 0
0B53 52 52 0
0B54 8 8 0
0B55 -1 4 0
0B56 -1 8 1
0B57 -1 4 1
0B58 2 2 1
0B59 4 4 1
0B5A -1 34 1
0B5B 14 14 1
0B5C 2 2 1
0B5D 10 10 1
0B5E 33 33 1
0B5F -1 5 0
0B60 -1 64 0
0B62 -1 17 1
0B63 -1 24 1
0B64 -1 7 1
0B65 -1 4 1
0B66 25 25 1
0B67 32 32 1
```

**Comment by mcpchen** [comment]
thanks ！i try it



发自我的iPhone


------------------ Original ------------------

**Comment by mcpchen** [comment]
Inventory is Unkonw&nbsp;





发自我的iPhone


------------------ Original ------------------

**Comment by mcpchen** [comment]
Thanks my en is very bad 。
can you spake Chinese，
哥们能看懂中文吗？好多事情请教，盼回复。



发自我的iPhone


------------------ Original ------------------

**Comment by alisonrag** [comment]
nope, im from brazil ^^

go to openkore\src\Network\Receive\Servertype0.pm

find:
```if (grep { $masterServer->{serverType} eq $_ } qw(iRO_Renewal)) {```
and replace with:
```if (grep { $masterServer->{serverType} eq $_ } qw(cRO)) {```

**Comment by mcpchen** [comment]
tkanks my en is very bad 。can you speak chines。哥们看的懂中文吗？好多问题请教



发自我的iPhone


------------------ Original ------------------

**Comment by alisonrag** [comment]
nope, im from brazil ^^

**Comment by mcpchen** [comment]
item is ok。 but Unkonwn switch 0B1B




发自我的iPhone


------------------ Original ------------------

**Comment by mcpchen** [comment]
you can update cRO Thankes



发自我的iPhone


------------------ Original ------------------

**Comment by alisonrag** [comment]
> item is ok。 but Unkonwn switch 0B1B 

this is the reason why this issue was opened

**Comment by mcpchen** [comment]
Is there a solution?

发自我的iPhone


------------------ Original ------------------

**Comment by alisonrag** [comment]
0AE2 implemented in #3001

**Comment by mcpchen** [comment]
thanks



发自我的iPhone


------------------ Original ------------------
